application/utils/logging.py

```diff
@@ -32,4 +32,4 @@ def add_service_name(logger, method_name, event_dict):  # pylint: disable=unused
         logger_factory=structlog.stdlib.LoggerFactory(),
         wrapper_class=structlog.stdlib.BoundLogger,
         cache_logger_on_first_use=True
```

Comment:
Small point, but when doing line by line args leave a trailing comma - it will help make future pull requests clearer 👍